owncloud/android
# [MDM] Making code more scalable

## Related Issues
App: https://github.com/owncloud/android/issues/3553
_____

## QA

Check list: https://github.com/owncloud/android/pull/3563#issuecomment-1072224933

1st test matrix: https://github.com/owncloud/android/pull/3563#issuecomment-1069350769
2nd test matrix: https://github.com/owncloud/android/pull/3563#issuecomment-1081716764

Reports:

- [X] (1) Deleted field https://github.com/owncloud/android/pull/3563#issuecomment-1073568001 [DONE]
- [X] (2) Empty field https://github.com/owncloud/android/pull/3563#issuecomment-1073585795 [DONE]
- [X] (3) UI test MDM flow becomes flaky https://github.com/owncloud/android/pull/3563#issuecomment-1073865807 [DONE]
